encoder/vaapiencoder_hevc.cpp

```diff
@@ -1022,16 +1023,7 @@ Encode_Status VaapiEncoderHEVC::setParameters(VideoParamConfigType type, Yami_PT
     case VideoParamsTypeAVC: {
```

Comment:
Besides this one, there are also other “AVC” structs and variables used in HEVC source code. Shall we change them all? Maybe we need choose a name which can covers both AVC and HEVC. I think we’d better think over it and change it later, instead of changing  in this version in a hurry.

Best Regards,
Xiaoxia Zhong

From: Xu Guangxin [mailto:notifications@github.com]
Sent: Monday, February 29, 2016 10:04 AM
To: 01org/libyami
Cc: Zhong, Xiaoxia
Subject: Re: [libyami] Add IDR interval setting to H.264 encoder (#447)


In encoder/vaapiencoder_hevc.cpp<https://github.com/01org/libyami/pull/447#discussion_r54363353>:

> @@ -1022,16 +1023,7 @@ Encode_Status VaapiEncoderHEVC::setParameters(VideoParamConfigType type, Yami_PT

>      case VideoParamsTypeAVC: {

could we change this to VideoParamsTypeHEVC?

—
Reply to this email directly or view it on GitHub<https://github.com/01org/libyami/pull/447/files#r54363353>.